index.bs

```diff
@@ -155,28 +179,31 @@ Parsing {#parsing}
 To <dfn abstract-op local-lt="parse header">obtain a response's embedder policy</dfn> given a
 [=response=] (|response|):
 
-1.  Let |policy| be "`unsafe-none`".
+1.  Let |parsed item| be the result of [=header list/getting a structured header=] with
+    "`Cross-Origin-Embedder-Policy`" and "`item`".
 
-2.  Let |header| be the result of [=header list/getting=] `Cross-Origin-Embedder-Policy` from
-    |response|'s [=response/header list=].
+2.  If |parsed item| is neither `failure` nor `null` and |parsed item|'s bare item is
+    "`require-corp`":
 
-3.  If |header| is not `null`:
+    1.  If |parsed item|'s parameters["report-to"] [=map/exists=] and it is a string, then set
+        |endpoint| to |parsed item|'s parameters["report-to"].
 
-    1.  Set |header| to the result of [=isomorphic decoding=] |header|.
+    2.  Return a new [=/embedder policy=] whose [=embedder policy/value] is |parsed item|'s
```

Comment:
This means that `COEPRO` is ignored in the presence of `COEP`. This is probably fine for the moment, since there's only one value, but would make it difficult to support enforcement of one value and reporting on another if we have multiple values in the future.

Perhaps add a note to that effect? I'm not sure it's worth adding complexity until we need it.

Reply:
I'd say COEPRO is ignored if its value is equal to or weaker than the COEP value. That rule is natural, and we'll want to keep the rule even when we introduce more values. What do you think?

Reply:
1. That's not the way CSP works.
2. What do we do with `COEP: require-corp\nCOEPRO: require-corp; report-to=whatever`? Should that not send reports?

Reply:
Hmm... so you prefer having two values and endpoints. I'll try to modify the logic.

Reply:
I think we will need to have two values and endpoints to support both enforce and report-only at the same time. I don't think that's actually relevant to any developer's life until we have multiple values (I have `require-cors` in the back of my head (because a value with a one-character difference to create radical behavioral differences is a great spelling option)), so I wouldn't mind explicitly punting on it. But doing it now certainly wouldn't be wasted effort.

Reply:
Fixed. Now the embedder policy concept has four members.
 - value
 - reporting endpoint
 - report only value
 - report only endpoint

Reply:
reporting endpoint / report only reporting endpoint is not set if value / report only value is not "require-corp" but it doesn't affect the behavior because we'll have nothing to report for "unsafe-none".